docs/readthedocs/source/doc/Nano/QuickStart/pytorch_openvino.md

```diff
@@ -1,6 +1,6 @@
 # BigDL-Nano PyTorch OpenVINO Acceleration Quickstart
 
-**In this guide we will describe how to apply OpenVINO Acceleration on inference pipeline with the APIs delivered by BigDL-Nano in 2 simple steps**
+**In this guide we will describe how to apply OpenVINO Acceleration on inference pipeline with the APIs delivered by BigDL-Nano in 3 simple steps**
```

Comment:
0 to 3 is 4 steps?

Reply:
I saw the Quickstart documentation for Chronos is calculated this way.
https://bigdl.readthedocs.io/en/latest/doc/Chronos/QuickStart/chronos-tsdataset-forecaster-quickstart.html